gradle/libs.versions.toml

```diff
@@ -0,0 +1,102 @@
+[plugins]
+avro = { id = "com.github.davidmc24.gradle.plugin.avro", version = "1.6.0" }
+analyze = { id = "com.faire.gradle.analyze", version = "1.0.9" }
+cutterslade-analyze = { id = "ca.cutterslade.analyze", version = "1.9.0" }
+
+[versions]
```

Comment:
Denne fila liker eg godt. Fint å kunne setje ein felles versjon for avhengnadar som går saman, og fint å ha alt av versjonering samla her.

Reply:
Ja. Liker den

Også det at man kan sette opp ting som brukes sammen ofte under bundles.